### DISCUSSION TITLE
Asking about Codespaces

### DISCUSSION THREAD

**Discussion by WebX-Crafters** [community/community#147741] [Codespaces] [answered]
### Select Topic Area

Question

### Body

Hi Everyone, 
I need your help in understanding the Codespaces platform

**Comment by lemorage** [comment]
I think u could check out the official docs here for more details: https://github.com/features/codespaces. They’ve done a great job explaining what it’s all about. Besides, if u know about [gitpod](https://www.gitpod.io/), they're sort of the same in essence.

But to put it simply, u could just regard Codespaces as just another computer you have access to (but on cloud). It runs on Ubuntu by default and comes with some handy tools pre-installed. Upon my case, I mostly use it for some projects I don't want to clone on my local—I’ll spin up a Codespace, set up all the dependencies I need for this proj, and get to work directly on it (so my disk space is saved huh). Super convenient!

**Comment by kareem1207** [comment]
Code spaces are just like virtual code editor, they are used if you don't have a editor nearby

**Comment by MohamedMagdy63** [comment, marked as answer]
GitHub Codespaces is a cloud-based development environment hosted by GitHub that provides instant, customizable, and fully-configured environments for software development. It is designed to improve productivity and eliminate the need for local setup.
